docs/diagrams/AddCommandSequenceDiagram.puml

```diff
@@ -57,6 +57,10 @@ AddCommand -> Model ++ MODEL_COLOR: hasPerson(person)
 deactivate Model
 AddCommand -> Model ++ MODEL_COLOR: addPerson(person)
 deactivate Model
```

Comment:
I'm not too sure how important it is, but perhaps this segment could be in an optional path based on whether `model.hasPerson()` or not?

Reply:
Hmmmmm good question! I guess the current implementation is good enough since `addPerson` is technically always called unless `model.hasPerson()` throws an exception. But then again, we were not taught how to represent exceptions in sequence diagrams.....

If we were to include optional paths, we might end up enclosing all subsequent operations after `model.hasPerson()` in an optional path, since these operations technically might not have the chance to be executed.

Tbh I'm not really sure about this too :sweat_smile:

Reply:
@Korochii Ohh, that's right, it would be more complicated that way. In that case, I think the current diagram works fine! 👍